.config/zsh/010.env.zsh##os.Darwin,e.zsh

```diff
@@ -0,0 +1,3 @@
+# macOS-specific brew paths
+PATH="/opt/homebrew/opt/trash-cli/bin:$PATH"
```

Comment:
You have to rename this file, that's it's not the same, after yadm removes the alternate markers. 

Name the beginning 011.env...